ramalama/common.py

```diff
@@ -322,7 +322,12 @@ def check_nvidia():
         command = ['nvidia-smi']
         run_cmd(command).stdout.decode("utf-8")
         os.environ["CUDA_VISIBLE_DEVICES"] = "0"
-        return "cuda"
+        command = [container_manager(), 'info', '--format', 'json']
+        result = run_cmd(command)
+        info = json.loads(result.stdout)
+        if 'host' in info and 'cdi' in info['host'] and 'devices' in info['host']['cdi']:
```

Comment:
Did you check to see if this works with Docker?

Reply:
Kinda, there is no host.cdi.devices in either docker or podman for me.

But I don't know about the case where host.cdi.devices has a value, is this portable for docker/podman.

Reply:
Do we need cdi?  If I run with `docker run --gpus all ...` it seems to run.
Also runs with podman.

Reply:
Currently seeing this on an Ampere aarch64 system with Nvidia, although we don't want to use Nvidia here, hence why it's not fully configured. I saw this on an x86_64 machine before also

Reply:
I would like to see if there is a non-null value here in the correct case before we merge, because I can't test the case for the correctly configured system, maybe the json is wrong